modules/core/src/testFixtures/java/org/apache/ignite/internal/testframework/IgniteTestUtils.java

```diff
@@ -947,6 +950,29 @@ public static void fillDummyFile(Path file, long fileSize) throws IOException {
         }
     }
 
+    /**
+     * Run the closure in the given executor, wait for the result and get it synchronously.
+     *
+     * @param executor Executor.
+     * @param closure Closure.
+     * @return Closure result.
+     */
+    public static <T> T runInExecutor(ExecutorService executor, Supplier<T> closure) {
+        Object[] arr = new Object[1];
```

Comment:
Curious whether it's a common practice to use single element array instead of AtomicReference or just a matter of case?